ampersand-multifield-view.js

```diff
@@ -138,12 +181,36 @@ var MultiFieldView = View.extend({
     View.prototype.remove.apply(this, arguments);
   },
 
+  reset: function() {
+    this.setValue(this.startingValue, true);
+    this.fields.forEach(function(field) {
+      if(field.reset) {
+        field.reset();
+      }
+    });
+  },
+
+  clear: function() {
+    // This won't unset field values, but will disable validation
+    this.setValue({}, true);
```

Comment:
Ampersand's clear, when implemented, calls `this.setValue(<empty>, true)` where `<empty>` is something like `''` or `false`.

Functionally, `this.shouldValidate = false` is the same, but I was trying to preserve how other field-views work in calling `setValue` in their `clear` functions. I'm not sure it's worth keeping. Thoughts?